pkg/types/aws/platform.go

```diff
@@ -101,6 +101,12 @@ type Platform struct {
 	//
 	// +optional
 	LBType configv1.AWSLBType `json:"lbType,omitempty"`
+
+	// IgnoreS3Deletion is an optional field that can be used to make the S3 deletion optional
+	// during bootstrap destroy.
+	// See: https://issues.redhat.com/browse/CORS-2661
```

Comment:
Let's not include a Jira link directly in the code (unless maybe a bug), but including in the commit message would be fine. It would be helpful to include in the commit message the motivation for this change.